src/resources/CRUDResource.ts

```diff
@@ -12,17 +12,22 @@ export enum CursorDirection {
 }
 
 /* Request */
-export interface CRUDPaginationParams {
+export type CRUDPaginationParams = {
     limit?: number;
     cursor?: string;
     cursorDirection?: CursorDirection;
-}
+};
 
 /* Response */
-export interface CRUDItemsResponse<A> {
+export type CRUDItemsResponse<A> = {
     items: A[];
     hasMore: boolean;
-}
+    totalHits?: number;
+};
+
+export type CRUDAOSItemsResponse<A> = CRUDItemsResponse<A> & {
+    totalHits?: number;
```

Comment:
do we need this? Cause we just added totalHits to CRUDItemsResponse 🤔

Reply:
Actually.. maybe we should not have added it to CRUDItemsResponse? 🤔

Reply:
Oh yeah, you are right. I added it to the wrong place and apparently forgot to remove it from there. Removing from `CRUDItemsResponse`.